### PR TITLE
Add Wireframe for User Settings Page

### PR DESCRIPTION
**Pull request by StephenODea54**
Ref #16 

Going to start implementing the API methods for Storage Providers, Teams, Projects, API Keys, etc.

I'm thinking that a "Projects" Section should live in the sidebar, but Storage Provider, Team Management, API Key stuff, etc should live within the User Settings realm:

image

### CODE REVIEW

File: src/components/layout/DashboardLayout.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const navigation = [
         links: [
             {
                 name: 'Settings',
-                href: '/settings',
+                href: '/settings/user-settings',
```
Comment (wetherc) [inline src/components/layout/DashboardLayout.tsx]: I think we can just leave this as settings, everything else will be scoped to the team or project (e.g., `/teams/<team uuid>/settings`)

File: src/components/settings/UserNavigation.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { useLocation } from 'react-router-dom';
+
+import { cn } from '@/lib';
+
+const TABS = [
+    { name: 'Settings', href: '/settings/user-settings' },
+    { name: 'API Keys', href: '/settings/api-keys' },
+    { name: 'Teams', href: '/settings/teams' },
```
Comment (wetherc) [inline src/components/settings/UserNavigation.tsx]: Should team settings live here? They're not really user-scoped

File: src/components/settings/UserNavigation.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { useLocation } from 'react-router-dom';
+
+import { cn } from '@/lib';
+
+const TABS = [
+    { name: 'Settings', href: '/settings/user-settings' },
```
Comment (wetherc) [inline src/components/settings/UserNavigation.tsx]: Can this just be the base `settings` slug?

**Review by wetherc — COMMENTED** [review]
Just some open questions, lemme know your thoughts!

File: src/components/layout/DashboardLayout.tsx (hunk 1 above)
Comment (StephenODea54) [reply src/components/layout/DashboardLayout.tsx]: That works

File: src/components/settings/UserNavigation.tsx (hunk 2 above)
Comment (StephenODea54) [reply src/components/settings/UserNavigation.tsx]: No doesn't have to we can put in the sidebar

File: src/components/settings/UserNavigation.tsx (hunk 3 above)
Comment (StephenODea54) [reply src/components/settings/UserNavigation.tsx]: Si

**Review by wetherc — APPROVED** [review]

**Pull request merged**